src/assets/Standards/CollectibleStandards/ERC721/ERC721Standard.test.ts

```diff
@@ -0,0 +1,71 @@
+import Web3 from 'web3';
+import HttpProvider from 'ethjs-provider-http';
+import { ERC721Standard } from './ERC721Standard';
+
+const MAINNET_PROVIDER = new HttpProvider(
+  'https://mainnet.infura.io/v3/341eacb578dd44a1a049cbc5f6fd4035',
```

Comment:
is there a way we can rely on a environment variable in the test suite so we don't abuse of this id?

Reply:
Agreed with this. I think we should investigate either using a local Ganache server to fake calls to the network, or use some kind of library that can intercept HTTP calls and cache the responses.

Reply:
Agree! If this is performing one single HTTP call mocking the request and response body should be enough right?

Reply:
Yeah I could probably just use nock to mock to this request yeah?

Reply:
Having some trouble figuring this out. Do either of you know how to effectively mock requests to our Web3 provider? I'm trying something like this:
```
nock('https://mainnet.infura.io/v3').post('/341eacb578dd44a1a049cbc5f6fd4035', {
  id: 21,
  jsonrpc: '2.0',
  method: 'eth_call',
  params: [
    {
      to: '0xf5de760f2e916647fd766b4ad9e85ff943ce3a2b',
      data:
        '0x6352211e00000000000000000000000000000000000000000000000000000000000068e1',
    },
    '0x97a82b',
  ],
});
```
Trying to [debug](https://github.com/nock/nock#debugging) the network activity in the tests but not seeing much useful in there. @wachunei or @mcmire thoughts?

Reply:
Yes I can help with this. What's the issue you're seeing? The request params and response body might have to be JSON objects.

Reply:
Well so I was trying to debug/log network activity with nock to see what the network requests looked like and I don't see the kinds of requests I'd expect:
![Screen Shot 2022-01-05 at 10 34 56 AM](https://user-images.githubusercontent.com/34557516/148254052-64572652-6f03-4c4c-ba52-f3a03062bdb4.png)

The shape I posted above was copy/pasted from the request body of a request logged in dev tools while using my local build. But I'm not sure if nock is intercepting these requests in the way we expect? If you're up for jumping on a call that would be great?

Reply:
Yup, happy to do that, I'll send you a DM.

Reply:
OK I think its finally working. But now I'm wondering if instead we should mock Web3... I would ask that we make that change in a future PR now though. @wachunei @mcmire what do you think?

Reply:
Will revisit this in a follow up PR.